src/events.ts

```diff
@@ -0,0 +1,194 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import * as express from 'express';
+import {
+  BackgroundEvent,
+  CloudFunctionsContext,
+  CloudFunctionsResource,
+  CloudEventsContext,
```

Comment:
I know we did not author this in this PR, but I think we should change:

`CloudEventsContext`
to
`CloudEventContext`

as we're talking about a single CloudEvent.

Reply:
Ack.